runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -255,35 +264,42 @@ NNFW_STATUS nnfw_session::load_model_from_modelfile(const char *model_file_path)
 
   std::string model_type = filename.substr(filename.size() - 7, 7);
```

Comment:
(out of this PR) Wow, `load_model_from_modelfile` gets model type in this way. `7` seemingly comes from len(".tflite") = len(".circle") = 7. Then, I guess `.tvn` is not working.

Again, `load_model_from_modelfile` is not public API. It seems just for internal use.